packages/ui/src/lib/tooltip/Tooltip.spec.ts

```diff
@@ -19,13 +19,43 @@
 import '@testing-library/jest-dom/vitest';
 
 import { render, screen } from '@testing-library/svelte';
-import { tick } from 'svelte';
+import { createRawSnippet, tick } from 'svelte';
 import { expect, test } from 'vitest';
 
 import Tooltip from './Tooltip.svelte';
 import { tooltipHidden } from './tooltip-store';
 
-test('tooltip is not empty string when tooltipHidden value false', async () => {
+const tip = 'test';
+
+test('Expect basic prop styling', async () => {
+  render(Tooltip, { tip: tip });
+
+  const element = screen.getByLabelText('tooltip');
+  expect(element).toBeInTheDocument();
+  expect(element).toHaveClass('bg-[var(--pd-tooltip-bg)]');
+  expect(element).toHaveClass('text-[var(--pd-tooltip-text)]');
+  expect(element).toHaveClass('border-[var(--pd-tooltip-border)]');
+  expect(element).toHaveClass('border-[1px]');
+});
+
+test('Expect basic slot styling', async () => {
+  render(Tooltip, {
+    tip: createRawSnippet(() => {
+      return {
+        render: (): string => 'test',
+      };
+    }),
```

Comment:
if you are not looking for the `test` string, you can probably just use `vi.fn()`

Reply:
Although you don't need the full / real snippet, these two tests are specifically testing the snippet code path and fall back to string tip with vi.fn(). So, could use something else, but didn't think it was worth mocking Snippet.